### ISSUE TITLE
ARIA: tab role example doesn't work

### ISSUE THREAD

**Issue by bendur** [mdn/content#30986]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

### What specific section or headline is this issue about?

Example

### What information was incorrect, unhelpful, or incomplete?

The example does not work with mouse or keyboard.

### What did you expect to see?

I expected to be able to select the other tabs and see their corresponding content.

### Do you have any supporting links, references, or citations?

**OS:** MacOS 14.1.2 (23B92)

**Browsers tested:** Firefox 120.0.1 (64-bit); Safari 7.1.2 (19616.2.9.11.12)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tab_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tab_role/index.md
* Last commit: https://github.com/mdn/content/commit/f6310f5cf34dfe4b5c0a45494a83b0a5ef9a6571
* Document last modified: 2023-11-14T20:39:27.000Z

</details>

**Comment by bsmth** [comment]
Thanks a lot for reporting! This is being tracked in https://github.com/mdn/content/issues/29325. I'm going to close this one so we can follow the progress in the other issue. Thanks :)
